### ISSUE TITLE
updateMatrixWorld required?

### ISSUE THREAD

**Issue by jaukia** [mrdoob/webvr-replayer#1]
Hi, thanks for the great plugin!

I had to add "camera.updateMatrixWorld( true );" after "camera.matrix.fromArray" in the play-function to make head moving work. Not sure if that is due to my setup or something with the replayer.

What I'm then seeing is that the head moves fine. Not sure if controllers are positioned correctly relative to the head though.

Also, didn't realise at first that you have to have window.controls and window.camera defined (i think) to have the replayer work. Documenting that might be useful, installation instructions might be handy as well.

**Comment by mrdoob** [comment]
> Also, didn't realise at first that you have to have window.controls and window.camera defined (i think) to have the replayer work.

Yeah, this extension is currently pretty hacky. Eventually it'll polyfill the WebVR directly.
